doc/api/_toc.md

```diff
@@ -47,5 +47,6 @@
 
 <div class="line"></div>
 
+* <a href="#" data-js-handler="offline-docs">Make available offline</a>
```

Comment:
Yeah, good point. The rationale behind making it always visible, is it might be valuable for visitors to get a hint of this functionality even though their browser don't support it. That said if we want to give them such a hint, we might as well be helpful enough to give some pointers as to which browsers have support.

Not sure we should render this link in the static files, could be programmatically inserted as you suggest.